### ISSUE TITLE
Add a circle to markers

### ISSUE THREAD

**Issue by gtrus21** [osmandapp/OsmAnd#18229]
### Describe the idea (required)

Please add an additional option for markers, which can be enabled separately - a circle of a given radius

![Screenshot_2023-09-26-08-47-27-863_net osmand plus-edit](https://github.com/osmandapp/OsmAnd/assets/69676792/47a365f5-b4c8-412f-ac8b-c6887422f47d)

### Tell us about the expected behaviour (required)

Add circle to markers 

### Tell us about alternatives you've considered (required)

Only radius-ruler.  But this is inconvenient for each individual marker

**Comment by gtrus21** [comment]
![Screenshot_2023-09-26-08-53-44-194_net osmand plus-edit](https://github.com/osmandapp/OsmAnd/assets/69676792/f3db3a88-7a7c-430a-812a-2266d4565c3b)

**Comment by DmitryAlexei** [comment]
For what purposes and to whom this option may be useful? Please describe the user story.

**Comment by gtrus21** [comment]
for example 
1) for searches: immediately see the boundaries of the search region and do not go beyond them.  
2) at any time see which poi / buildings / objects are within the radius of each marker

**Comment by vshcherb** [comment]
Yes that's probably a duplicate issue cause people want to draw area around a certain point and that's how markers could be used.

**Comment by vshcherb** [comment]
Merged https://github.com/osmandapp/OsmAnd/issues/13440
